lib/v2/theverge/router.js

```diff
@@ -1,3 +1,3 @@
 module.exports = (router) => {
-    router.get('/', require('./index'));
+    router.get('/:hub?', require('./index'));
```

Comment:
Please update the docs and maintainer.js as well

Reply:
Done, updated.